### ISSUE TITLE
Screen readers don't read the link label for document links

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#3243]
We recently implemented a fix to add the document type to what screen readers read for document links, but now 'download [document type] is all it reads - it doesn't read the visual text label.

Here's what it looks like in ANDI. I confirmed with a screen reader that it only reads "Download PDF" instead of "Provide contact information (ONRR-4444)  - download PDF"

<img width="890" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd0l0Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0b8faec25eb8d3a16309e96f759599e5780da213/image.png" alt="image.png" />

**Comment by AlexandracmONRR** [comment]
aria-label just shows "Download PDF" but doesn't read the document's name.

**Comment by lpgoldstein** [comment]
In preview. Will release this sprint
